.gitignore

```diff
@@ -2,3 +2,4 @@
 /build
 /test_smis/*.ass
 __pycache__
+.vscode
```

Comment:
Could you revert this?

Reply:
Yes, this is automatically added by my IDE setting.